apps/api/src/campaign-application/campaign-application.controller.spec.ts

```diff
@@ -18,6 +18,7 @@ describe('CampaignApplicationController', () => {
   let controller: CampaignApplicationController
   let service: CampaignApplicationService
   let personService: PersonService
+  const { isAdmin } = require('../auth/keycloak')
```

Comment:
Use the ES6 import  `import {isAdmin} from `../auth/keycloak`, over the require() statement